angular-rails-app/config/database.yml

```diff
@@ -33,8 +33,10 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
-  database: heatseek_test
+  adapter: sqlite3
+  database: db/test.sqlite3
+  pool: 5
+  timeout: 5000
```

Comment:
Sadly, it does not. It's just the config.

You still have to go through the `rake db:migrate`, However, I've wrapped that up in the `entrypoint_test.sh` file. So as you add more tests, the command to run them can be added there.